modules/ROOT/pages/mule-upgrade-tool.adoc

```diff
@@ -1,15 +1,17 @@
 = Mule Upgrade Tool
 
-*Latest version: 1.1.0*
+*Latest version: 1.1.1*
 
-include::release-notes::partial$mule-upgrade-tool/mule-upgrade-tool-1.1.0.adoc[tag=intro]
+include::release-notes::partial$mule-upgrade-tool/mule-upgrade-tool-1.1.1.adoc[tag=intro]
```

Comment:
This should be a general intro that is independent of versions. We don't want to maintain versioning here. 

If we're going to use an include, let's see if we can do this differently.

Reply:
See PR https://github.com/mulesoft/docs-release-notes/pull/6060/files where we will need to sort this out.